masim.c

```diff
@@ -189,7 +193,7 @@ static void do_seq_rw(struct access *access, int batch)
 {
 	struct mregion *region = access->mregion;
 	char *rr = region->region;
-	size_t offset = access->last_offset;
+	uint64_t offset = access->last_offset;
```

Comment:
Questions about `size_t` size of 64bit machines apply to above three changes, too.